### PR TITLE
Add hard upstream timeouts for maps

### PR DESCRIPTION
**Pull request by bsstoner**
We did this for the dictionary spice a few weeks ago and it eliminated a ton of the fallbacks. I want to try it on maps and see if this will help at all.

```
ab -c2 -n100 https://duckduckgo.com/local.js?q=41%20perry%20street%2C%20new%20york

va-xl1
Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        3    4   0.6      4       9
Processing:   605  760 112.0    736    1143
Waiting:      605  760 112.0    736    1143
Total:        609  764 112.2    740    1147

Percentage of the requests served within a certain time (ms)
  50%    740
  66%    787
  75%    811
  80%    824
  90%    913
  95%   1037
  98%   1121
  99%   1147
 100%   1147 (longest request)


sing-xl1
Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        2    3   0.4      3       5
Processing:   714  774  45.9    760     919
Waiting:      714  774  45.9    760     918
Total:        717  777  45.9    763     921

Percentage of the requests served within a certain time (ms)
  50%    763
  66%    780
  75%    790
  80%    808
  90%    841
  95%    893
  98%    920
  99%    921
 100%    921 (longest request)


eur-xl1
Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        3    3   0.3      3       5
Processing:   688  883 177.2    857    1934
Waiting:      688  883 177.2    857    1934
Total:        691  887 177.2    861    1937

Percentage of the requests served within a certain time (ms)
  50%    861
  66%    896
  75%    933
  80%    955
  90%   1049
  95%   1151
  98%   1724
  99%   1937
 100%   1937 (longest request)


cali-xl1
Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        4    5   0.2      5       6
Processing:   532  657 117.0    635    1309
Waiting:      532  657 117.0    635    1309
Total:        537  662 117.0    639    1314

Percentage of the requests served within a certain time (ms)
  50%    639
  66%    688
  75%    725
  80%    734
  90%    793
  95%    905
  98%    982
  99%   1314
 100%   1314 (longest request)
```

---
https://duck.co/ia/view/maps_maps

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Spice/Maps/Maps.pm [hunk 1]
```diff
@@ -20,6 +20,11 @@ my @all_triggers = ('directions');
 # - "directions from leeds to skipton uk" (matches "skipton uk")
 my $directions_qr = qr/^(\w+\s)?directions.*\bto\b/;
 
+spice upstream_timeouts => +{ connect => '100ms',
+                              send => '100ms',
+                              read => '1000ms'
+                          }
```
Comment (duendex) [inline lib/DDG/Spice/Maps/Maps.pm]: I think you need to add a semicolon here

Comment (bsstoner) [inline lib/DDG/Spice/Maps/Maps.pm]: nice catch, thank you

**Comment by bsstoner** [comment]
Sorry, I should have tested that this morning. Added the semicolon and all tests pass now.

**Comment by duendex** [comment]
LGTM

**Pull request merged**